ts/src/config.ts

```diff
@@ -115,16 +115,35 @@ export interface Config extends GoogleAuthOptions {
   // For testing with startLocal() only.
   localProfilingPeriodMillis?: number;
 
-
   // Debugging information for startLocal will be recorded every
   // localLogPeriodMillis milliseconds.
   // For testing with startLocal() only.
   localLogPeriodMillis?: number;
 
-
   // Duration of time profiles collected when using startLocal().
   // For testing with startLocal() only.
   localTimeDurationMillis?: number;
+
+  // List of directories recursively searched for *.js.map files. Defaults to
+  // process.cwd().
+  //
+  // The profiler uses these files to re-map the source file paths in the
+  // profiles. The most common use case of having a source map is an application
+  // or dependency written in TypeScript: the source file paths that
```

Comment:
The text here says "or dependency", but that's not correct since node_modules is not searched yet?

Reply:
Done.